tools/editor/project_manager.cpp

```diff
@@ -871,8 +871,8 @@ ProjectManager::ProjectManager() {
 
 	HBoxContainer *top_hb = memnew( HBoxContainer);
 	vb->add_child(top_hb);
-	TextureFrame *logo = memnew( TextureFrame );
-	logo->set_texture(theme->get_icon("LogoSmall","EditorIcons"));
+	//TextureFrame *logo = memnew( TextureFrame );
+	//logo->set_texture(theme->get_icon("LogoSmall","EditorIcons"));
```

Comment:
I would, but I just followed what @reduz did at 8baa85d6b4428b3fcbcd7ef160e608471ea7606b
I think so it's better to remove these lines.
I will update this PR. :)